networkx/drawing/tests/test_pydot.py

```diff
@@ -28,7 +28,12 @@ def build_graph(self, G):
 
     def assert_equal(self, G1, G2):
         assert_true( sorted(G1.nodes())==sorted(G2.nodes()) )
-        assert_true( sorted(G1.edges())==sorted(G2.edges()) )
+        if G1.is_directed():
+            assert_true(sorted(G1.edges()) == sorted(G2.edges()))
+        else:
+            e1 = sorted(map(sorted, G1.edges()))
```

Comment:
I see that you are continuing the previous assumption that says "nodes are sortable". Should we make this assumption more explicit. In Python 3, this requires that nodes be of the same type as well. So all int, all string, etc. Also, this seems like an idiom for how to test if two edge sets are equal independent of the graph type. Do we repeat this idiom in many places in NetworkX? If so, perhaps a function would be better.